ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -450,3 +455,7 @@
 (define-event ToggleShowRouteLine [routename]
   {:path [:transit-visualization :compare :show-route-lines]}
   (update app routename not))
+
+(define-event ToggleShowNoChangeRoutes []
+  {:path [:transit-visualization :show-no-change-routes?]}
+  (not app))
```

Comment:
Some readers might be distracted by the symbol naming - `not` evaluated using `:show-no-change-routes?` istead of `app`, but I suppose this is the simplest way to do this :)

Reply:
This might be the shortest way to do this but not sure if it is the simplest or best.
But other toggle methods are similar, so I copied the functionality from there.